src/pages/leejiyon/Login/Login.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './login.scss';
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: '',
+      pw: '',
+    };
+  }
+
+  onChangeInput = e => {
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  render() {
+    const validation = this.state.id.includes('@') && this.state.pw.length > 4;
+    return (
+      <div className="LoginJY">
+        <main className="LoginBox">
+          <h1 className="title">Westagram</h1>
+          <div className="login">
+            <input
+              className="loginInputs loginId"
+              name="id"
+              value={this.state.id}
+              onChange={this.onChangeInput}
+              type="text"
+              placeholder="전화번호, 사용자 이름 또는 이메일"
+            />
+            <input
+              className="loginInputs loginPw"
+              name="pw"
+              value={this.state.pw}
+              onChange={this.onChangeInput}
+              type="password"
+              placeholder="비밀번호"
+            />
+          </div>
+
+          <Link to="/mainjy">
+            {validation ? (
+              <button className="loginBtn blueBackgroundColor">로그인</button>
+            ) : (
+              <button className="loginBtn ">로그인</button>
+            )}
```

Comment:
```suggestion
            {validation ? (
              <button className={validation ? "loginBtn" : "loginBtnBlueBackgroundColor"}>
              로그인
              </button>
     }
```

className 약간 손보셔서 button 태그 전체가 아닌, className만 삼항연산자로 바꾸시면 더욱 좋을 거 같습니다!

Reply:
확인했습니다 !

Reply:
```
<button
    className={`loginBtn ${
      validation ? 'blueBackgroundColor' : null
   }`}
>
```
이렇게 수정했습니다